internal/provider/provider.go

```diff
@@ -101,20 +108,28 @@ func azureProvider() *schema.Provider {
 				Description: "The Client Secret which should be used. For use When authenticating as a Service Principal using a Client Secret.",
 			},
 
-			// Managed Service Identity specific fields
-			"use_msi": {
+			"skip_provider_registration": {
```

Comment:
We don't support provider registration, so no need to expose this.

Reply:
This is supported in the track2 SDK: https://github.com/Azure/terraform-provider-azapi/blob/681dac0a2c6b5fdfe0b6d936a5a259e28c88165f/vendor/github.com/Azure/azure-sdk-for-go/sdk/azcore/arm/client_options.go#L30-L31

Reply:
Can we add `skip_provider_registration` to the doc?